Fonts/PTSenif-Bold/specimen_files/grid_12-825-55-15.css

```diff
@@ -0,0 +1,181 @@
+/* Notes about grid:
+Columns:      12
+Grid Width:   825px
+Column Width: 55px
+Gutter Width: 15px
+------------------------------- */
+
+.section { margin-bottom: 18px;
+}
+
+.section:after { content: ".";display: block;height: 0;clear: both;visibility: hidden;
+}
+
+.section { *zoom: 1;
+}
+
+.section .firstcolumn,
+.section .firstcol { margin-left: 0;
+}
+
+/* Border on left hand side of a column. */
+.border {
+  padding-left: 7px;
+  margin-left: 7px;
+  border-left: 1px solid #eee;
+}
+
+/* Border with more whitespace, spans one column. */
+.colborder {
+  padding-left: 42px;
+  margin-left: 42px;
+  border-left: 1px solid #eee;
+}
+
+/* The Grid Classes */
+.grid1,
+.grid1_2cols,
+.grid1_3cols,
+.grid1_4cols,
+.grid2,
+.grid2_3cols,
+.grid2_4cols,
+.grid3,
+.grid3_2cols,
+.grid3_4cols,
+.grid4,
+.grid4_3cols,
+.grid5,
+.grid5_2cols,
+.grid5_3cols,
+.grid5_4cols,
+.grid6,
+.grid6_4cols,
+.grid7,
+.grid7_2cols,
+.grid7_3cols,
+.grid7_4cols,
+.grid8,
+.grid8_3cols,
+.grid9,
+.grid9_2cols,
+.grid9_4cols,
+.grid10,
+.grid10_3cols,
+.grid10_4cols,
+.grid11,
+.grid11_2cols,
+.grid11_3cols,
+.grid11_4cols,
+.grid12 { margin-left: 15px;float: left;display: inline; overflow: hidden;
+}
+
+.width1,
+.grid1,
+.span-1 { width: 55px;
+}
+
+.width1_2cols,
+.grid1_2cols { width: 20px;
+}
+
+.width1_3cols,
+.grid1_3cols  { width: 8px;
+}
+
+.width1_4cols,
+.grid1_4cols  { width: 2px;
+}
+
+.input_width1 { width: 49px;
+}
+
+.width2, .grid2, .span-2 { width: 125px; }
+
+.width2_3cols,
+.grid2_3cols  { width: 31px;
+}
+
+.width2_4cols,.grid2_4cols  { width: 20px; }
+.input_width2 { width: 119px; }
+
+.width3, .grid3, .span-3 {width: 195px;}
```

Comment:
Codacy found an issue: [Expected single space after "{" of a single-line block (block-opening-brace-space-after)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6241956)